spec.html

```diff
@@ -18677,6 +18703,17 @@ <h1>Runtime Semantics: EvaluateBody</h1>
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-arrow-function-definitions-runtime-semantics-namedevaluation">
+      <h1>Runtime Semantics: NamedEvaluation</h1>
+      <p>With parameter _name_.</p>
+      <emu-grammar>ArrowFunction : ArrowParameters `=&gt;` ConciseBody</emu-grammar>
+      <emu-alg>
+        1. Let _closure_ be the result of evaluating this |ArrowFunction|.
+        1. Perform SetFunctionName(_closure_, _name_).
```

Comment:
```suggestion
        1. Perform ! SetFunctionName(_closure_, _name_).
```

Reply:
I'd like to keep this without !, because there's still no resolution for #486 and it matches the majority of existing SetFunctionName callers.

Reply:
Totally fine for this PR; but I do see #486 as resolved, so I'll be happy to put up a followup PR that converts all SetFunctionName callsites to use `!` or `?` as needed.